package.json

```diff
@@ -18,6 +18,7 @@
     "prettier": "prettier --write 'src/**/*.js' 'demo/src/**/*.js'"
   },
   "dependencies": {
+    "lodash": "^4.17.10",
```

Comment:
Can you switch to https://www.npmjs.com/package/lodash.debounce?
(I don't know whether tree shaking is there yet).

Reply:
using the `cherryPick` option will do? https://github.com/insin/nwb/blob/master/docs/Configuration.md#cherrypick-string--arraystring